OpenSim/Common/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ OpenSimAddLibrary(
     )
 
 if(WIN32)
-    add_dependencies(osimCommon Simbody_CONFIG_check)
+    add_dependencies(osimCommon
+        Simbody_CONFIG_check Copy_Simbody_DLLs)
```

Comment:
Maybe just add a comment why Simbody_CONFIG_check is Windows only. Thanks!

Reply:
Done.